code/messaging/src/main/java/com/adobe/marketing/mobile/MessagingPushBuilder.java

```diff
@@ -89,7 +97,7 @@ private static String createChannelAndGetChannelID(final MessagingPushPayload pa
             final NotificationManager notificationManager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
             final String channelIdFromPayload = payload.getChannelId();
 
-            // if a channel from the payload is not null and if a channel exists for the channel ID from the payload, use the same channel ID.
+            // if a channel from the payload
```

Comment:
can this comment be clarified?

Reply:
I removed them by mistake. Keeping the comment back